app/src/main/java/com/practicum/playlistmaker/PlayerActivity.kt

```diff
@@ -42,6 +51,30 @@ class PlayerActivity : AppCompatActivity() {
         val playerCountry = findViewById<TextView>(R.id.player_country_name)
         val playerArtworkImage = findViewById<ImageView>(R.id.player_image)
 
+        mediaPlayer = MediaPlayer()
+        mediaPlayer.setDataSource(previewUrl)
+        mediaPlayer.prepareAsync()
+
+        playerPlayTrack = findViewById<ImageButton>(R.id.player_play_track)
+        playerTimePlnw = findViewById<TextView>(R.id.player_time_plnw)
+        handler = Handler()
```

Comment:
Этот конструктор помечен как deprecated, нужно использовать следующий конструктор:
[Handler](https://developer.android.com/reference/android/os/Handler#Handler(android.os.Looper))([Looper](https://developer.android.com/reference/android/os/Looper) looper)
```suggestion
        handler = Handler(Looper.getMainLooper())
```